From 7e9bcf54c1de9b810abf77b61f5fab891b2d76f5 Mon Sep 17 00:00:00 2001
From: Kiran Rudramuni <kchitrik@codeaurora.org>
Date: Thu, 15 Oct 2015 18:42:20 +0530
Subject: [PATCH 01/40] Fix: Avoid reboot issue if battery is not present

This change will force OTG to conrol via PHY in place of PMIC

Signed-off-by: eInfochips Team <ei-qc8064@einfochips.com>
---
 arch/arm/mach-msm/board-8064.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-msm/board-8064.c b/arch/arm/mach-msm/board-8064.c
index 63beaa0..253134c 100644
--- a/arch/arm/mach-msm/board-8064.c
+++ b/arch/arm/mach-msm/board-8064.c
@@ -911,7 +911,7 @@ static int phy_init_seq[] = {
 
 static struct msm_otg_platform_data msm_otg_pdata = {
 	.mode			= USB_OTG,
-	.otg_control		= OTG_PMIC_CONTROL,
+	.otg_control		= OTG_PHY_CONTROL,/*OTG_PMIC_CONTROL,*/
 	.phy_type		= SNPS_28NM_INTEGRATED_PHY,
 	.pmic_id_irq		= PM8921_USB_ID_IN_IRQ(PM8921_IRQ_BASE),
 	.power_budget		= 750,
-- 
1.9.1

